designer/client/pages/LandingPage/ChooseExisting.tsx

```diff
@@ -27,6 +27,7 @@ export class ChooseExisting extends Component<Props, State> {
 
   componentDidMount() {
     formConfigurationApi.loadConfigurations().then((configs) => {
+      console.log(configs);
```

Comment:
please remove any console.logs or use the logger if you wish to merge back to our repo